--- dosfstools-4.0/src/Makefile.am~	2016-05-06 01:19:07.000000000 +0000
+++ dosfstools-4.0/src/Makefile.am	2016-05-08 17:35:16.172875745 +0000
@@ -34,12 +34,12 @@
 mkfs_fat_SOURCES  = mkfs.fat.c msdos_fs.h $(mkfs_common_sources)
 mkfs_fat_CPPFLAGS = -I$(srcdir)/blkdev
 mkfs_fat_CFLAGS   = $(AM_CFLAGS) $(UDEV_CFLAGS)
-mkfs_fat_LDFLAGS  = $(UDEV_LIBS)
+mkfs_fat_LDADD  = $(UDEV_LIBS)
 
 testdevinfo_SOURCES  = testdevinfo.c $(mkfs_common_sources)
 testdevinfo_CPPFLAGS = -I$(srcdir)/blkdev
 testdevinfo_CFLAGS   = $(AM_CFLAGS) $(UDEV_CFLAGS)
-testdevinfo_LDFLAGS  = $(UDEV_LIBS)
+testdevinfo_LDADD  = $(UDEV_LIBS)
 
 
 if COMPAT_SYMLINKS
